src/plays/quote-generator/quoteGenerator.css

```diff
@@ -0,0 +1,99 @@
+.play-area {
```

Comment:
Based on the class name you have added above, please use that for all your style here.

Reply:
Class name changed from play-area to rand-quote-gen